src/email.rs

```diff
@@ -86,6 +86,23 @@ https://crates.io/confirm/{}",
     send_email(email, subject, &body)
 }
 
+/// Attempts to send a crate owner invitation email. Swallows all errors.
+///
+/// Whether or not the email is sent, the invitation entry will be created in
+/// the database and the user will see the invitation when they visit
+/// https://crates.io/me/pending-invites/.
+pub fn send_owner_invite_email(email: &str, user_name: &str, crate_name: &str) {
+    let subject = "Please confirm crate ownership";
```

Comment:
No problem, please be as picky as you'd like :) And I can see how that original subject sounded like it may refer to an existing ownership confirmation. I've updated it to "Crate ownership invitation".